src/components/sidenav/sidenav.js

```diff
@@ -299,8 +299,9 @@ function SidenavDirective($mdMedia, $mdUtil, $mdConstant, $mdTheming, $animate,
               ])
               .then(function() {
                 // Perform focus when animations are ALL done...
-                if (scope.isOpen) {
-                  focusEl && focusEl.focus();
+                if (scope.isOpen && focusEl) {
+                  focusEl.focus();
+                  focusEl.triggerHandler('focus');
```

Comment:
I don't understand why this is necessary. Invoking `focus` on an element should always trigger any event listeners to be called. @DevVersion can you explain?